Goose.API/Authorization/AuthorizationUtils.cs

```diff
@@ -61,5 +61,21 @@ public static void ThrowErrorIfAllFailed(this AuthorizationResult authorizationR
 
             throw new HttpStatusException(StatusCodes.Status403Forbidden, errorMsg);
         }
+
+        public static IList<ObjectId> RolesOfUser(params PropertyUser[] propertyUsers)
+        {
+            IList<ObjectId> roles = new List<ObjectId>();
+
+            if (propertyUsers is null) return roles;
+
+            foreach (var propertyUser in propertyUsers)
+            {
+                if (propertyUser is not null)
+                    roles = roles.ConcatOrSkip(propertyUser.RoleIds);
```

Comment:
warum verwendest du nicht roles.AddRange()?

Performancetechnisch macht es in diesem Fall nicht viel aus, da ein User wahrscheinlich nie mehr als 2 Rollen hat. Bei größeren Listen wäre der Code hier aber sehr ineffizient

Reply:
das musst du im Phillip seinem PR fragen ist von ihm o: 
generell ich hab das meiste hier ist von Phillip hab ja nur den merge gemacht. 
Nur alles mit messages ist von mir

Reply:
> warum verwendest du nicht roles.AddRange()?
> 
> Performancetechnisch macht es in diesem Fall nicht viel aus, da ein User wahrscheinlich nie mehr als 2 Rollen hat. Bei größeren Listen wäre der Code hier aber sehr ineffizient

Guter Punkt, ich hatte die Funktion `AddRange()` in dem Moment wo ich es implementiert habe nicht auf dem Schirm. Da es aber keinen unterschied macht (niemals wird jemand mehr als 100/1000 Rollen habe), würde ich es so lassen, außer du bestehst darauf, dass es angepasst wird.

Reply:
Warum ich übrigens `ConcatOrSkip` implementiert habe war einerseits, da der Übergabe Parameter auch mal null sein könnte. `AddRange` würde da glaube ich eine Exception werfen.

Reply:
ob Concat oder AddRange ist mir egal. Mich hat der Grund für die Implementation interessiert, da AddRange im Normalfall die erste Wahl wäre.